src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -117,20 +349,15 @@ static void verify(CSharpCompilation comp)
                 var model = comp.GetSemanticModel(tree);
                 var method = model.GetDeclaredSymbol(nodes.OfType<MethodDeclarationSyntax>().Single());
```

Comment:
>.GetSymbol<NamedTypeSymbol>() [](start = 63, length = 29)

I am not sure why bother going through SemanticModel and then do this to get to private symbols.

Reply:
> I am not sure why bother going through SemanticModel and then do this to get to private symbols.

This one was marked as resolved, but it doesn't look addressed.

---
In reply to: [374864023](https://github.com/dotnet/roslyn/pull/40994#discussion_r374864023) [](ancestors = 374864023)